src/App.js

```diff
@@ -1,13 +1,54 @@
 import React from 'react';
+import FormInput from '../src/components/TodoComponents/TodoForm';
+import ListItems from '../src/components/TodoComponents/TodoList';
+
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      input: '',
+      myTodoList: []
+    };
+  }
+
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const myObj = {
+      task: this.state.input,
+      id: Date.now(),
+      completed: false
+    };
+
+    if(this.state.input === ''){
+      alert('please enter something');
```

Comment:
I really like this here! Great job making sure you don't have a bunch of empty tasks!